### PR TITLE
Education

### PR DESCRIPTION
**Pull request by lee2smooth03**
Hey Patrick, can you review these?
I need to practice on my machine

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -23,11 +23,11 @@ <h1>Chatham Apprentice Program Application</h1>
               <legend>Demographic Info</legend>
```
Comment (Bentley912) [inline public/index.html:23]: Change Title to Appropriate Title Please

File: public/js/education.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+$(document).ready(function(){
```
Comment (Bentley912) [inline public/js/education.js:1]: You didn't need to create a new JS file. Simply change the button and add it to the form JS

File: public/js/form.js [hunk 3]
```diff
@@ -30,11 +30,12 @@ $(document).ready(function(){
     })
```
Comment (Bentley912) [inline public/js/form.js]: What did you do here?

File: public/js/education.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    $('.alert').hide();
+    $('.eduForm').submit(function(){
```
Comment (Bentley912) [inline public/js/education.js:3]: You dont need this code if you are using the saveData() function.

**Review by Bentley912 — CHANGES_REQUESTED** [review]
Thanks for this. There are some small changes to the code including Title of the page. Right now, is it saving to sessionStorage? Was it verified that we are saving it somewhere?

**Review by Bentley912 — APPROVED** [review]

**Review by Bentley912 — CHANGES_REQUESTED** [review]
Didnt mean to Approve. Previous Changes still need to happen.

**Comment by lee2smooth03** [comment]
Ok...I got red letters in my terminal last time I ran node server.js

On Apr 19, 2018 5:40 PM, "Patrick Bentley" <notifications@github.com> wrote:

*@Bentley912* requested changes on this pull request.

Didnt mean to Approve. Previous Changes still need to happen.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Bentley912/cap_app/pull/17#pullrequestreview-113787202>,
or mute the thread

**Pull request closed without merging**